packages/app-content-pages/server/server.js

```diff
@@ -11,6 +11,11 @@ const handleNextRequests = app.getRequestHandler()
 app.prepare().then(() => {
   const server = new http.Server((req, res) => {
     app.setAssetPrefix(assetPrefix)
+
+    if (!dev) {
```

Comment:
Does this carry over to staging? Is `dev` really indicative of all non-production environments?

Reply:
`dev` here is shorthand for `process.env.NODE_ENV !== production`. The `NODE_ENV` environment variable is set locally, and in production it's set in the `deployment.tmpl` file

Reply:
How does caching work on staging env?

Reply:
I assume there is no staging server for this app, right? Feel free to ignore this if that's the case but dev isn't just `!= 'production'` in most setups.

Reply:
there isn't. that said, would it make more sense to actually have ENABLE_CACHE as an env var as well?